docs/rules/no-static-typos.md

```diff
@@ -0,0 +1,59 @@
+# Prevent casing typos when declaring static propTypes, contextType and defaultProps
+
+Ensure no casing typos were made declaring static class properties
+
+**Fixable:** This rule is automatically fixable using the `--fix` flag on the command line.
```

Comment:
Ok, I will remove it. 

I was actually not aware of this policy. So only stylistic issues should be fixed? Adding removing lines, spaces, semi-colons?

FWIW, The actual help for `--fix` actually says the following: `This option instructs ESLint to try to fix as many issues as possible.`

Considering the words "as many issues as possible" - I thought that every rule which we know how to fix should be auto fixable.